src/main/java/it/pagopa/selfcare/pagopa/backoffice/security/PagopaAuthenticationStrategy.java

```diff
@@ -82,18 +58,14 @@ public JwtAuthenticationToken authenticate(JwtAuthenticationToken authentication
 
         final Collection<GrantedAuthority> authorities;
         try {
-            authorities = authoritiesRetriever.retrieveAuthorities();
+            authorities = null;
```

Comment:
Why wrap this collection assignation in a try-catch block? Can be removed?

Reply:
good point. Resolved in this [PR](https://github.com/pagopa/pagopa-selfcare-ms-backoffice-backend/pull/275)